packages/core/src/routes/sso-connector/index.ts

```diff
@@ -29,9 +29,10 @@ export default function singleSignOnRoutes<T extends AuthedRouter>(...args: Rout
     router,
     {
       id: tenantId,
+      libraries: { ssoConnectors: ssoConnectorLibrary },
       queries: { ssoConnectors },
       libraries: {
-        ssoConnector: { getSsoConnectorById, getSsoConnectors },
+        ssoConnectors: { getSsoConnectorById, getSsoConnectors },
```

Comment:
can desctructure two `libraries` here?

Reply:
should remove the above libraries.  Looks like a rebase issue

Reply:
updated